antarest/study/web/output_blueprint.py

```diff
@@ -571,8 +573,10 @@ def get_output_variables_view(
             renewable_id,
             st_storage_id,
         )
-        content = dataframe.to_json(orient="split", index=False)
-        return Response(content=content, media_type="application/json")
+        if isinstance(view, pd.DataFrame):
+            content = view.to_dict(orient="split", index=False)
+            return Response(content=to_json(content), media_type="application/json")
+        return view
```

Comment:
The transformation to response should be implemented here, not in the service layer